WordPressKit/ServiceRemoteWordPressComREST.m

```diff
@@ -26,6 +26,7 @@ - (instancetype)initWithWordPressComRestApi:(WordPressComRestApi *)wordPressComR
     self = [super init];
     if (self) {
         _wordPressComRestApi = wordPressComRestApi;
+        _wordPressComRESTAPI = wordPressComRestApi;
```

Comment:
Once all the concrete `WordPressComRestApi` usages will have been replaced, this init will take an `id<WordPressComRESTAPIInterfacing>` input, effectively decoupling `ServiceRemoteWordPressComREST` (Objective-C) from `WordPressComRestApi` (Swift) via `WordPressComRESTAPIInterfacing` (Objective-C).

This is the model implemented in https://github.com/Automattic/spm-abstraction-layer-demo which should allow us to organize the code in a Swift package that Xcode can compile.